ResuMe/urls.py

```diff
@@ -15,7 +15,7 @@
 """
 from django.contrib import admin
 from django.urls import path
-from resumeApp import views
+from Feed import views
 
 urlpatterns = [
     path('admin/', admin.site.urls),
```

Comment:
Following the previous suggestion, I would replace the next line with `path('', views.Feed, name='Feed')`

Reply:
Ok, I changed it according to what you mentioned